src/embedded_flash/syscall.rs

```diff
@@ -210,6 +213,117 @@ fn find_slice<'a>(
     Err(StorageError::OutOfBounds)
 }
 
+struct PartitionSlice {
+    pub ptr: usize,
+    pub len: usize,
```

Comment:
Removed for `RangeInclusive`, see below. In case we like the struct more than the new type, since this moved to `helper.rs`, it would still need to be either `pub` or get a (trivial) `new`.